src/harness/testcases/WINNF_FT_S_PAT_testcase.py

```diff
@@ -173,18 +172,20 @@ def test_WINNF_FT_S_PAT_1(self, config_filename):
     config = loadConfig(config_filename)
     num_tests = len(config)
     test_pass_threshold = 0.999 #percentage of passed test expected
-    num_passed_tests = 0.0
+    num_passed_tests = 0
     num_tests = len(config)
     max_fail_num = int ((1-test_pass_threshold) * num_tests)
-    num_failed_tests = 0.0
-    num_invalid_tests = 0.0
-    for request in  config:
+    num_failed_tests = 0
+    num_invalid_tests = 0
+
+    for request, test_num in enumerate(config):
```

Comment:
Please check the documentation on how enumerate works and run the code before posting for re-review.

Reply:
Oops. That's fixed. Thanks